editor/src/scripts/parse-print-analysis.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint no-console: "off" */
+import * as Path from 'path'
+import * as FS from 'fs'
```

Comment:
path and fs are usually imported as lower case in node scripts, when I read `FS.readFileSync` in the code I thought you are using a third party library that wraps `fs`

Reply:
+1 to this, as silly as the convention seems

Reply:
I'll be doing this in a subsequent PR as I've already made a bunch of changes to this there.